articles/migrations/guides/extensibility-node8.md

```diff
@@ -1,57 +1,69 @@
 
+
+
+
 ## Summary of the migration
 
-The Webtask runtime powering the following Auth0 features utilize Node.js v4:
+The Webtask runtime powering the following Auth0 features utilize Node 4:
 
 * Rules
 * Hooks
 * Custom database connections
 * Custom social connections
 
-Due to the end of long-term support (LTS) for Node.js v4, we will be migrating the Webtask runtime to use Node.js v8.
+
+If you do not use any of the extensibility features mentioned above, you are not affected by this migration. **Additionally, your tenant will automatically be upgraded to use the Node 8 runtime on April 30, 2018.** This will ensure that any future extensibility code you author will be running on a secure runtime.
+
+Due to the end of long-term support (LTS) for Node 4, we will be migrating the Webtask runtime to use Node 8.
 
 As part of this migration, the Auth0 development team has performed extensive testing to detect any breaking changes proactively.
 
-However, there may be behavioral changes as a result of this migration. As such, we have provided a migration switch that allows you to control the migration of your environment to the new Webtask runtime using Node.js v8.
+However, there may be behavioral changes as a result of this migration. As such, we have provided a migration switch that allows you to control the migration of your environment to the new Webtask runtime using Node 8.
 
-## How to enable the Node.js v8 runtime
+## How to enable the Node 8 runtime
 
 :::warning
-The Extensibility panel, that this section refers to, will be made available to our cloud customers on **April 17th, 2018**.
+The Extensibility panel, that this section refers to, will be made available to our cloud customers on **April 17, 2018**.
 :::
 
-Node 8 can be enabled through the new Extensibility panel on the [Advanced Tenant Settings](${manage_url}/#/tenant/advanced) page of the Dashboard. This panel will be visible once the new Node 8 runtime is made available on 17 April 2018.
+Node 8 can be enabled through the new Extensibility panel on the [Advanced Tenant Settings](${manage_url}/#/tenant/advanced) page of the Dashboard. This panel will be visible once the new Node 8 runtime is made available on April 17, 2018.
 
 ![Runtime toggle](/media/articles/migrations/node-runtime1.png)
 
-![Runtime toggle options](/media/articles/migrations/node-runtime1.png)
+![Runtime toggle options](/media/articles/migrations/node-runtime2.png)
 
 ::: warning
-Changing the runtime may break your existing Rules, Hooks, and Custom Database/Social Connections. We recommend that you first switch your development tenant, test your setup, and switch your production tenant only if there are no breaking changes.
+Changing the runtime may break your existing Rules, Hooks, and Custom Database/Social Connections. We recommend that you first switch to your development tenant, test your setup, and switch to your production tenant only if there are no breaking changes.
```

Comment:
I'm not sure the add `to`'s make sense? I think we're trying to say that you first should first update your dev tenant to use Node 8.

```
Changing the runtime may break your existing Rules, Hooks, and Custom Database/Social Connections. We recommend that you first update your development tenant to use Node 8 and test your setup there. Then, switch to your production tenant and upgrade to Node 8 only if there are no breaking changes.
```